packages/docs/PropTables/PopoverPropTable.tsx

```diff
@@ -48,16 +48,16 @@ const popoverProps: Prop[] = [
     description: 'If set to true, the Popover will have the same width as its anchor element.',
   },
   {
-    name: 'offsetX',
+    name: 'skidding',
     types: 'number',
     defaultValue: '0',
-    description: 'Determines the popover offset on the X axis.',
+    description: 'Determines the offset along the anchorElement.',
```

Comment:
🍹 I'd specify the direction of the offset, eg for skidding `Determines the horizontal offset along the anchorElement`
and for distance I'd say `Determines the vertical offset along the anchorElement`. I think it's a bit more clear

Reply:
In this case, there is no concept of horizontal/vertical since a Popover can be placed in any direction (top, right, etc).

![image](https://user-images.githubusercontent.com/2752665/91886595-2aa13180-ec4f-11ea-989f-3815fc8c219d.png)